kernel/selinux/rules.c

```diff
@@ -170,9 +170,7 @@ static int get_object(char *buf, char __user *user_object, size_t buf_sz,
 // reset avc cache table, otherwise the new rules will not take effect if already denied
 static void reset_avc_cache()
 {
-#if ((KERNEL_VERSION(4, 14, 0) <= LINUX_VERSION_CODE) &&                       \
-     (LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 163))) ||                     \
-	(LINUX_VERSION_CODE < KERNEL_VERSION(4, 9, 337))
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 10, 0)) && !defined(KSU_COMPAT_USE_SELINUX_STATE)
```

Comment:
we can define a macro standalone, such as:

```
#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5, 10, 0)) || defined(KSU_COMPAT_USE_SELINUX_STATE)
#define KSU_XXXX
#endif 
```

Reply:
https://github.com/tiann/KernelSU/pull/402/commits/6bf07bf51559512439290da07af1381faa63de14 done.